bin/logstash.lib.sh

```diff
@@ -100,7 +100,8 @@ setup_java() {
       if [ -x "$LS_JAVA_HOME/bin/java" ]; then
         JAVACMD="$LS_JAVA_HOME/bin/java"
         if [ -d "${LOGSTASH_HOME}/${BUNDLED_JDK_PART}" -a -x "${LOGSTASH_HOME}/${BUNDLED_JDK_PART}/bin/java" ]; then
-          echo "WARNING: Using LS_JAVA_HOME while Logstash distribution comes with a bundled JDK."
+          BUNDLED_JDK_VERSION=`cat JDK_VERSION`
+          echo "WARNING: Logstash distribution comes with a bundled JDK(${BUNDLED_JDK_VERSION}), which is a known-working and tested java distribution. You are recommended to use the bundled JDK."
```

Comment:
```
WARNING: Logstash distribution comes with a bundled JDK(Temurin-11.0.14.1+1), which is a known-working and tested java distribution. You are recommended to use the bundled JDK.
```
needed to read this twice to grasp, maybe it's just me but would rather have @karenzone double check wording

Reply:
What about something like:
> Logstash comes bundled with the recommended JDK(${BUNDLED_JDK_VERSION}), but is overridden by the version defined in `LS_JAVA_HOME`.  Consider clearing `LS_JAVA_HOME` to use the bundled JDK.  

Here's my thinking: 

- We have to be careful about making explicit recommendations that might not be appropriate for everybody.
- A good message ties back to the root cause (in this case `LS_JAVA_HOME` being set) and gives user some suggestions for resolving (in this case clearing `LS_JAVA_HOME`).

Reply:
I switch back to the original wording

Reply:
Hey, @andsel.  I liked the improvements you were making in the messaging. Are we running into a character limit or some other issue?

Reply:
>  Are we running into a character limit or some other issue?

No, I don't think. The line is almost 200 characters wide, which is long, probably close to a limit of characters use din modern terminals. 
If you feel it's too long, we can split in 2 halves.